src/main/webapp/EventForm.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<meta charset="UTF-8" />
+<title>Event Form</title>
+<link rel="stylesheet" href="style.css" />
+<div id="content">
+  <div id="location-container"></div>
+  <form action="/api/events" method="post">
+    <h1>Enter an Event</h1>
+
+    <p>Name:</p>
+    <textarea name="name" placeholder="Busan International Boat Show"></textarea>
+
+    <p>City ID</p>
+    <textarea name="cityId" placeholder="1"></textarea>
```

Comment:
@gferioli0418 @NzingaHolloman @parth-vora @TengkuHanaShafikah 
Similar concept here as in the other comment except you'd be working with cities instead of countries